src/Error/ErrorTrap.php

```diff
@@ -126,7 +126,10 @@ public function handleError(
         try {
             // Log first incase rendering or event listeners fail
             $logger->logMessage($error->getLabel(), $error->getMessage());
-            $this->dispatchEvent('Error.handled', ['error' => $error]);
+            $event = $this->dispatchEvent('Error.handled', ['error' => $error]);
```

Comment:
If this event prevents rendering, we should call it `beforeRender`.

Reply:
I am ok with `Error.beforeRender` as well if folks think it is better. It has the benefit of sharing terminology and behavior with controller and view events which is nice.